iceoryx2-ffi/cxx/include/iox2/unique_port_id.hpp

```diff
@@ -13,9 +13,15 @@
 #ifndef IOX2_UNIQUE_PORT_ID_HPP
 #define IOX2_UNIQUE_PORT_ID_HPP
 
+#include "iox/optional.hpp"
 #include "iox2/internal/iceoryx2.hpp"
 
+#include <array>
```

Comment:
Initially tried to use `iox::vector` but didn't work out how to set the bytes properly without `vector::empty` returning false.